R/utilities.R

```diff
@@ -101,3 +101,24 @@ loadPages <- function (object, url) {
   }
   return(result)
 }
+
+# We only support a specific format returned by KissMetrics, so we'll use
+# lubridate::fast_strptime instead of parse_date_time.
+# Take a character vector, and if we think it is a time that can be parsed by
+# format we'll return a vector of times, if it can't be converted (any of the
+# non-NA elements fail to parse) we return the original vector
+try_convert_time <- function(char_vector, formats = "%Y-%m-%d %H:%M:%S") {
+  result <- char_vector
+
+  timezone <- Sys.getenv("FROM_TIMEZONE")
```

Comment:
`KISSMETRICS_CONFIGURED_TIMEZONE`? and probably postfixed with `_ZONENAME` or `_ZONECODE`?